### PR TITLE
[1LP][RFR] handle ntp server set up issues during appliance pre-configuration

### PR DESCRIPTION
**Pull request by izapolsk**

### CODE REVIEW

**Comment by mshriver** [comment]
We might not really need this, but after talking with @izapolsk it seems this is the error we hit if we try to make API calls when the database service is being restarted.

**Comment by jawatts** [comment]
@izapolsk Looks like travis is failing on isort.

**Comment by izapolsk** [comment]
@jawatts, travis failed because of auto reformatting some file.

**Comment by jawatts** [comment]
> @jawatts, travis failed because of auto reformatting some file.

@izapolsk did it modify isort.cfg?

File: cfme/utils/appliance/__init__.py [hunk 1]
```diff
@@ -1111,13 +1112,9 @@ def set_ntp_sources(self, log_callback=None):
...
 
-        ntp_servers = {
-            'ntp_server_{n}'.format(n=i + 1): time_servers[i]
-            for i in range(0, len(time_servers))
-        }
-        logger.info('Setting NTP servers from config file: %s', ntp_servers)
+        logger.info('Setting NTP servers from config file: %s', time_servers)
 
-        self.server.settings.update_ntp_servers(ntp_servers)
+        self.update_advanced_settings({'ntp': {'server': time_servers}})
```
Comment (mshriver) [inline cfme/utils/appliance/__init__.py]: :man_facepalming: Not sure how/why I didn't find this in the REST docs.

**Comment by mshriver** [comment]
Travis failure from isort, there is no change to requirements or the pre-seed in this PR, so I'm ignoring that failure. #8487 moves away from isort.

**Pull request merged**